src/main.py

```diff
@@ -1084,13 +1074,15 @@ def _simple_primary_switch(self, limit, new_primary, is_dead):
                 #
                 # The easy way succeeded.
                 #
-                logging.info('Simple primary switch succeeded.')
+                logging.info('ACTION. Simple switch primary to {} succeeded'.format(new_primary))
+                self.checks['primary_switch'] = 0
+                self._reset_simple_primary_switch_try()
                 return True
             else:
                 return False
 
     def _rewind_from_source(self, is_postgresql_dead, limit, new_primary):
-        logging.info("Starting pg_rewind")
+        logging.info("ACTION. Starting pg_rewind")
```

Comment:
I think it is not an `ACTION`.
Here we are just beginning to process a rewind branch, with a lot of checks.
`Starting pg_rewind` intent should be logged right before calling `pg_rewind` executable.